src/style.css

```
@@ -0,0 +1,29 @@
+body {
+  margin: 0;
+  background-color: black;
+}
+
+.blue {
+  width: 350px;
+  box-sizing: border-box;
+  margin: 20px 20px 40px;
+  border: 1px solid #e75551;
+  background-color: #308dc0;
+}
+
+.yellow {
+  margin: 10px;
+  border: solid;
+  border-width: 1px;
+  border-color: #e75551;
```

```suggestion
  border: 1px solid #e75551;
```
why in blue it's short but there not?